draft-ietf-mops-streaming-opcons.md

```diff
@@ -458,9 +458,11 @@ In order to avoid these types of situations, which can potentially affect all th
 
 There are many reasons why path characteristics might change suddenly, for example,
 
-* "cross traffic" that traverses part of the path, especially if this traffic is "inelastic", and does not, itself, respond to indications of path congestion.
+* routing changes, which can happen in normal operation, especially if the new path now includes path segments that are more heavily loaded, offer lower total bandwidth, provide different MTU sizes that change the overall Path MTU size, or simply cover more distance.
 
-* routing changes, which can happen in normal operation, especially if the new path now includes path segments that are more heavily loaded, offer lower total bandwidth, or simply cover more distance.
+* changes in business relationships between economic entities that, together, consititute an end-to-end path, so that a packet that might have been carried over a certain path is now being carried over a path that is partially or entirely disjoint from the previous path. The overall effect might be indistinguishable from the effect of routing changes, in the previous bullet.
```

Comment:
Is this different from routing changes?  It sounds to me like this is describing a routing change in a transit provider driven by a business change instead of some technical reason?  Slightly confused by this bullet.

All the rest seems good tho.

Reply:
@GrumpyOldTroll - I think I understand your confusion (I know a lot about people being confused by text I provide :upside_down_face:).

I think what I should be saying here is that path characteristics may change 
- because the path segment topology changes, often because of routing changes, or 
- because the "cross traffic" that is sharing part or all of the path changes.

Plus additional commentary, but does that make sense?

Reply:
@SpencerDawkins will update and merge.